README.md

```diff
@@ -27,3 +28,15 @@ about the current recursive call.
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. You do not need to prove that the invariant is correct. Add
 your answer to this markdown file.
+----When using this invariant, we can test any number. For example, if we want to test 4, we just look at the previous two elements in the array and add them. We assume this is correct before we add the two elements, which would be at index -1 and index, since the index of the array is base 0.
+
+Citations:
+Citations: 
+https://www.w3schools.com/js/js_arrays.asp Used above site on a review of syntax for javascript arrays. 
+Used chatgpt to tell me what the issues in my code were: " Your Fibonacci function has a few issues. Here's a revised version that correctly generates Fibonacci numbers and appends them to an array:
```

Comment:
So the code you submitted was written by ChatGPT?

Reply:
I asked how to overall save values to an array without needing to recreate the array everytime, ChatGPT told me I needed to make a function inside the already given fib(n) function. The highlighted section was me showing chatGPT told me the errors in my code on why it was not appending to the array correctly. ChatGPT illustrated the issues in my code and may have given a counterexample, but this was not fully written my ChatGPT shown when I pushed my code to another online "compiler" it wrote out my array stored too many elements, chatGPT did not find this error which is why I troubleshooted and needed to solve this without chatGPT.

Reply:
So what part of the code was written by you?

Reply:
if(x<=0)
else if(x==1) {
else{
            fibArray.push(recurFib(x-1)); - this line was changed, since before it was calling recursion for each x-1 and x-2
            let idx1 = fibArray.length - 1;
          fibArray[idx1] + fibArray[idx1-1]
The above lines are the ones I either wrote or modified since the array was having issues. The logic behind, which is shown from one of the first pull requests, was asked if it looked correct so far, which the TA suggested not having as many recursive calls. I asked about syntax by both the TA and chatGPT on how to push since before I was using the wrong syntax since this is some of my first times coding in javascript. If needed I can change anything you feel I did not write myself.

Reply:
I did not include the commented out section, but if needed to show how I debugged to solve the issue with my array, please let me know. Thank you.

Reply:
This sounds like most of it was not written by you. It would be great if you could submit something that is mostly your work.

Reply:
I may have explained wrongly, and for that I apologize. However, this code that passes the tests is written by me, yes I consulted ChatGPT when I ran into issues but even with the code it may suggest, the array does not save properly and does not pass the tests since it uses recursion incorrectly.

Reply:
If you still request changes, I understand, I just ask for clarification on why this is not mostly my work?